compiler-core/src/language_server/engine.rs

```diff
@@ -198,10 +198,12 @@ where
             // Check current filercontents if the user is writing an import
             // and handle separately from the rest of the completion flow
             // Check if an import is being written
-            if let Some(value) = this.import_completions(src, &params, module) {
+            let src_line_numbers = LineNumbers::new(src.as_str());
```

Comment:
Sorry, I'm confused- what's `src` here and why is it different from `module.code`?

Reply:
src was the raw buffer content vs module.code being the buffer content based on the last compile. For finding the current cursor point and phrase around the cursor, using the direct src seems more correct since we don't want there to be weird completion location drift if the code is left in a noncompiling state for too long

Reply:
Is it possible for them to be different now that we have fault tolerant compilation?

Reply:
I think there are still edge cases where it is possible mainly in cases where the currently typed thing fails to parse/lex which will happen a lot for completions where the user just types the dot

```
import gleam/io

pub fn main() {
  io.
}
```